RoomsEyeView/Logic/vtkSlicerRoomsEyeViewModuleLogic.cxx

```diff
@@ -423,13 +443,162 @@ void vtkSlicerRoomsEyeViewModuleLogic::BuildRoomsEyeViewTransformHierarchy()
     vtkErrorMacro("BuildRoomsEyeViewTransformHierarchy: Invalid MRML scene");
     return;
   }
+  vtkSlicerBeamsModuleLogic* beamsLogic = vtkSlicerBeamsModuleLogic::SafeDownCast(this->GetModuleLogic("Beams"));

```

Comment:
Where I said you can access logic classes dynamically I meant this. No need to define them as members I think.